### ISSUE TITLE
`block.resize` must return itself instead of `true`

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo#3540]
Currently, the `block` in `malloc` returns `true` on `resize`. It doesn't make much sense and is not helpful. I suggest it to return itself (the `block`).

**Comment by yegor256** [comment]
@maxonfjvipon WDYT? Any objections?

**Comment by maxonfjvipon** [comment]
@yegor256 no. It also it should be renamed to `resized`

**Comment by maxonfjvipon** [comment]
@yegor256 I'm trying to implement `resized` object and I faced an interesting situation. The piece of code in `malloc-tests` looks like: `(m.resized 6).size.eq 6`. 
Here I'm resizing block and trying to get the size of the block.
But I noticed that block is not being resized and here's why:

```
[id] > allocated
  [] > size
  [size] > resized /allocated
```

Here `resized` object has attribute `size` and `allocated` object has attribute `size`. When I write  `(m.resized 6).size.eq 6` I assume that I take `allocated.size` object after resizing. But I got `allocated.resized.size` attribute.

WDYT about such case? Is there a way we can prevent or predict such situations because they're quite confusing for unprepared developers
